sale_commission/wizard/wizard_settle.py

```diff
@@ -63,7 +63,7 @@ def action_settle(self):
             date_to_agent = self._get_period_start(agent, date_to)
             # Get non settled invoices
             agent_lines = agent_line_obj.search(
-                [('invoice_date', '<', date_to_agent),
+                [('invoice_date', '<', date_to),
```

Comment:
So you need to take next period date to get current ones settlements? Looks a bit weird.